Traducir.Api/Traducir.Api.csproj

```diff
@@ -8,8 +8,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.3" />
     <PackageReference Include="MiniProfiler.AspNetCore.Mvc" Version="4.0.0-alpha9-00206" />
-    <PackageReference Include="StackExchange.Exceptional.AspNetCore" Version="2.0.0-rc2.12" />
     <PackageReference Include="Simple.Migrations" Version="0.9.19" />
+    <PackageReference Include="StackExchange.Exceptional.AspNetCore" Version="2.0.0-rc2.12" />
```

Comment:
you are still touching this file (you can see that [on the diff](https://github.com/g3rv4/Traducir/pull/34/files)).

one thing you can do is, while you're on your `pr18` branch, is `git checkout master Traducir.Api/Traducir.Api.csproj`

Once you do that, you'll see you have changes on this file. Add, commit and push them :)